### PR TITLE
add detail on types of talks

### PR DESCRIPTION
**Pull request by jgadsden**
**Summary** :  
This pull request adds some detail on what types of talks we would like to see
It also changes the slides link to the Bristol page rather than the London page

**Description for the changelog** :  
add detail on types of talks

**Other info** :

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -12,22 +12,32 @@ postal-code:
 
 # Welcome to OWASP Bristol Chapter 
 
-The OWASP Bristol Chapter  is  run by [Katy Anton](mailto:katy.anton@owasp.org) and [Craig Francis](mailto:craig.francis@owasp.org) .
+The OWASP Bristol Chapter is run by [Katy Anton][katy], [Craig Francis][craig] and [Jon Gadsden][jon].
```
Comment (craigfrancis) [inline index.md:15]: Should the email addresses be removed?

Comment (craigfrancis) [reply index.md:15]: Oh, interesting, just noticed the footer links, not seen those before.

**Review by craigfrancis — APPROVED** [review]
Seems fine, thanks Jon :-)

File: index.md (hunk 1 above)
Comment (jgadsden) [reply index.md:15]: yes, it is a common technique now, and makes the raw markdown much more readable. Also keeps the line lengths shorter, better all round

**Pull request merged**